prism/src/main/java/org/apache/falcon/resource/AbstractSchedulableEntityManager.java

```diff
@@ -111,16 +111,29 @@ protected synchronized void scheduleInternal(String type, String entity, Boolean
         }
     }
 
+    /**
+     * Validates the parameters wheather SLA is supported or not.
+     *
+     * @param entityType currently two entityTypes are supported Process and Feed
+     * @param entityName name of the entity
+     * @param start startDate from which SLA is to be looked at
+     * @param end endDate upto which SLA is to be looked at.
+     * @param colo colo in which entity is to be looked into
+     * @throws FalconException if the validation fails
+     * **/
+
     public static void validateSlaParams(String entityType, String entityName, String start, String end,
                                          String colo) throws FalconException {
         EntityType type = EntityType.getEnum(entityType);
-        if (type != EntityType.FEED) {
+        LOG.info("EntityName:" + entityName);
+        if (!Arrays.asList(EntityType.FEED.toString().toLowerCase(), EntityType.PROCESS.toString().toLowerCase()).
```

Comment:
this is the reason I used Arrays.asList it should be 
 type != EntityType.FEED && type != EntityType.PROCESS
...its more readable  :)

Anyway found a better after our discussion type.isScheduleable ...